chain/client/src/metrics.rs

```diff
@@ -135,6 +135,26 @@ pub(crate) static VALIDATORS_CHUNKS_EXPECTED_BY_SHARD: Lazy<IntGaugeVec> = Lazy:
     .unwrap()
 });
 
+pub(crate) static VALIDATORS_CHUNK_ENDORSEMENTS_PRODUCED_BY_SHARD: Lazy<IntGaugeVec> =
+    Lazy::new(|| {
+        try_create_int_gauge_vec(
+            "near_validators_chunk_endorsements_produced_by_shard",
+            "Number of chunk endorsements produced by a chunk validator",
```

Comment:
Can you rename and / or update the description to reflect the subtlety of how this is computed?

Reply:
done